src/components/styles/ProjectPreview.styled.ts

```diff
@@ -0,0 +1,63 @@
+import styled, { keyframes, FontType } from 'styled-components'
+import { fontTypeCss } from '../../styles/index.styled'
+
+const FadeIn = keyframes`
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+`
+
+export const PreviewText = styled.div<{ hasAnimation: boolean; fontType: FontType }>`
+  ${fontTypeCss}
+  color: ${(props) => props.theme.palette.common.white};
+  @keyframes fadeIn {
+    0% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+  animation-name: ${FadeIn};
+  animation-delay: 1s;
+  animation-duration: 1.5s;
+  animation-fill-mode: both;
+  ${(props) => {
```

Comment:
consider 
```
${(props) =>
    props.hasAnimation &&
    css`animation-name: ${FadeIn};
  animation-delay: 1s;
  animation-duration: 1.5s;
  animation-fill-mode: both;`}
``` 
instead and removing line 24-27

Reply:
acb0a0e9ab69f76346e7d80a15addbffef9a7514